site/docs/v1/tech/tutorials/integrate-expressjs.md

```diff
@@ -0,0 +1,153 @@
+
+{% include docs/integration/_install-fusionauth.md %}
+
+## Create a User and an API Key
+
+{% include docs/integration/_add-user.md language=page.language %}
+
+## Configure FusionAuth
+
+Next, you need to set up FusionAuth. This can be done in different ways, but we’re going to use the {{page.language}} client library. The below instructions use `npm` on the command line, but you can use the client library with an IDE of your preference as well.
+
+First, make a directory:
+
+```shell
+mkdir setup-fusionauth && cd setup-fusionauth
+```
+
+Now, copy and paste the following file into `package.json`.
+
+```javascript
+{% remote_include https://raw.githubusercontent.com/FusionAuth/fusionauth-example-client-libraries/main/typescript/package.json %}
+```
+
+Now you need to install the dependencies in `package.json`.
+
+```shell
+npm install
+```
+
+Then copy and paste the following file into `setup.js`. This file uses the [FusionAuth API](/docs/v1/tech/apis/) to configure an Application and more to allow for easy integration. 
+
+```javascript
+{% remote_include https://raw.githubusercontent.com/FusionAuth/fusionauth-example-client-libraries/main/typescript/setup.js %}
+```
+
+Then, you can run the setup script.
+
+{% include _callout-note.liquid content="The setup script is designed to run on a newly installed FusionAuth instance with only one user and no tenants other than `Default`. To follow this guide on a FusionAuth instance that does not meet these criteria, you may need to modify the above script. <br><br> Refer to the [Typescript client library](/docs/v1/tech/client-libraries/typescript) documentation for more information." %}
+
+```shell
+fusionauth_api_key=YOUR_API_KEY_FROM_ABOVE npm run setup
+```
+
+If you are using PowerShell, you will need to set the environment variable in a separate command before executing the script.
+
+```shell
+$env:fusionauth_api_key='YOUR_API_KEY_FROM_ABOVE'
+npm run setup
+```
+
+If you want, you can [log into your instance](http://localhost:9011) and examine the new Application the script created for you.
+
+## Create Your {{page.technology}} Application
+
+Now you are going to create a {{page.technology}} application. While this section uses a simple {{page.technology}} application, you can use the same steps to integrate any {{page.technology}} application with FusionAuth.
+
+First, make a directory.
+
+```shell
+mkdir ../setup-expressjs && cd ../setup-expressjs
+```
+
+Create a `package.json` file with the following contents to set up the dependencies.
+
+```json
+{% remote_include https://raw.githubusercontent.com/FusionAuth/fusionauth-example-node/master/package.json %}
+```
+
+Now, change into the `setup-expressjs` directory and install the needed packages.
+
+```shell
+cd setup-expressjs && npm install
+```
+
+You are going to create some files in different directories, so pay attention to the final directory structure that you should have after completing these steps. 
```

Comment:
Do we have this directory tree in a consistent place across all the other tutorials?

Reply:
This is something specific to [our node example repo](https://github.com/FusionAuth/fusionauth-example-node)

Reply:
Ah, sorry for being unclear. Some of our integration tutorials have the tree here, some have the tree after you create the application, and some don't have them at all.

See https://fusionauth.io/docs/v1/tech/tutorials/integrate-ruby-rails (doesn't have it), https://fusionauth.io/docs/v1/tech/tutorials/integrate-python-flask (has it just before starting to test the auth flow) and here.

I want the location and presence of the tree to be somewhat standardized. One option would be to say 'put the tree picture after the "test the auth flow" header' if the tree is small enough. Rails generates so many files that I hesitate to put it in that one, however.

But that's what I meant.

Reply:
After discussing this last week, I've created issue #2191